src/components/banner/Banner.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Icon from '../icon';
+import { BannerType, BannerIcon } from './BannerUtils';
+
+type BannerProps = {
+  /** Optional CSS class name */
+  className?: string;
+  /** If true, close icon will be placed to the right */
+  canClose?: boolean;
+  /** Function to call on close action */
+  onClickClose?: () => void;
```

Comment:
Can we simplify as "onClose" ? wdyt

Reply:
So I move from `onClickClose` to `onClose`, and from `onClickAction` to `onAction`?